src/Microsoft.AspNet.TestHost/project.json

```diff
@@ -7,6 +7,7 @@
     },
     "dependencies": {
         "Microsoft.AspNet.Hosting": "1.0.0-*",
+        "Microsoft.AspNet.WebSockets.Protocol": "1.0.0-*",
```

Comment:
This dependency will cause a build order issue on our CI (HttpAbstractions -> Hosting -> Servers -> WebSockets -> Hosting). I think there's a simpler way to implement it that doesn't require the dependency anyways.  Because the data is never sent out over an actual network, we don't even have to serialize the web socket messages down to a Stream, we can just pass the full messages back and forth between the 'client' and 'server' ends of the websocket. You'd need to change your reader/writer buffer to store full messages rather than just bytes.